components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -745,6 +752,14 @@ interface WritableHistoryConnection : ReadableHistoryConnection {
      * @param visitTimestamp The timestamp of the visit to delete, in MS since the unix epoch
      */
     fun deleteVisit(url: String, visitTimestamp: Long)
+
+    /**
+     * Records an accepted autocomplete match, recording the query string,
+     * and chosen URL for subsequent matches.
+     *
+     * @param result SearchResult String
```

Comment:
Nit: can you update the @param here now that this takes the search string and url?

Reply:
Sorry about this. Done!